### PR TITLE
[ADMIN-Editorial] Bug fixes for dmse objects

### PR DESCRIPTION
**Pull request by jpradocueva**
I have created a new PR with the changes that are needed in the DMSE objects to resolve the current validation issues: https://github.com/OpenMobileAlliance/objects-lwm2m/issues/95

Before merging this PR the corresponding CRs need to be created against the affected Enablers and presented to DMSE not later than June 16th; approval date.
I am assuming that the TS is not updated. If this is not the case, then Objects and DDF.xml MAY need an upate.

### CODE REVIEW

File: DDF.xml [hunk 1]
```diff
@@ -216,7 +216,7 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
         <Owner>OMA</Owner>
         <Source>0</Source>
         <Ver>1.0</Ver>
-        <DDF>LWM2M_LOCKWIPE-v1_0_1.xml</DDF>
+        <DDF>LWM2M_LOCKWIPE-v1_0_2.xml</DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
         <TS>http://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_1-20181003-A/OMA-TS-LWM2M_LockWipe-V1_0-20180615-A.pdf</TS>
```
Comment (seanmcilroy29) [inline DDF.xml]: LOCKWIPE TS will require updating to incorporate the changes to v1_0_1 - Did you want to update the links in this PR?

File: DDF.xml [hunk 2]
```diff
@@ -230,7 +230,7 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
         <Owner>OMA</Owner>
         <Source>0</Source>
         <Ver>1.0</Ver>
-        <DDF>LWM2M_Software_Management-v1_0.xml</DDF>
+        <DDF>LWM2M_Software_Management-v1_0_1.xml</DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
         <TS>http://www.openmobilealliance.org/release/LWM2M_SWMGMT/V1_0-20180301-A/OMA-TS-LWM2M_SwMgmt-V1_0-20180301-A.pdf</TS>
```
Comment (seanmcilroy29) [inline DDF.xml]: SwMgmt TS will require updating to incorporate the changes to v1_0_1 - Did you want to update the links in this PR?

**Comment by seanmcilroy29** [comment]
@jpradocueva - All the associated TS links in these Objects will require updating, did you want to apply the updates to this PR or in another?

**Review by jmudge — CHANGES_REQUESTED** [review]
- The Legal Disclaimers need to be updated e.g. in LWM2M_DevCapMgmt-v1_0_1.xml & LWM2M_Portfolio-v1_0_1.xml .
- LWM2M_DevCapMgmt-v1_0_1.xml it is suggested that line 103 is updated to "9..15: reserved"

**Comment by jpradocueva** [comment]
@seanmcilroy29 @jmudge we will have to keep this PR here until the new TS's are published. At the moment the validator detects that the TS are not available.

**Comment by seanmcilroy29** [comment]
@jpradocueva - bug-fix branch now validates with no errors

**Review by jmudge — APPROVED** [review]

**Pull request merged**